akka-actor-tests/src/test/scala/akka/actor/SchedulerSpec.scala

```diff
@@ -240,11 +240,12 @@ trait SchedulerSpec extends BeforeAndAfterEach with DefaultTimeout with Implicit
         "stop continuous scheduling if the task throws exception" taggedAs TimingTest in {
           EventFilter[Exception]("TEST", occurrences = 1).intercept {
             val count = new AtomicInteger(0)
-            collectCancellable(scheduleAdapter.schedule(Duration.Zero, 20.millis, () => {
-              val c = count.incrementAndGet()
-              testActor ! c
-              if (c == 3) throw new RuntimeException("TEST") with NoStackTrace
-            }))
+            collectCancellable(scheduleAdapter.schedule(Duration.Zero, 20.millis,
+                () => {
```

Comment:
So it's four spaces now?

Reply:
See https://github.com/mdedetrich/akka-apache/pull/2#discussion_r1002689305